operators/test/e2e/kb/association_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kb
+
+import (
+	"testing"
+
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/test"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/test/elasticsearch"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/test/kibana"
+)
+
+// TestCrossNSAssociation tests associating ElasticSearch running in a different namespace.
```

Comment:
Nit: `s|ElasticSearch|Elasticsearch|`

I would tend to be more explicit in the comment: ".. associating Elasticsearch and Kibana running in different namespaces."